src/TwigFactory.php

```diff
@@ -51,10 +51,10 @@ public function create( array $loaders ): Twig_Environment {
 	}
 
 	public function newWikiPageLoader( WikiContentProvider $provider ) {
-		if ( empty( $this->config['loaders']['wiki'] ) ) {
+		if ( !$this->config['loaders']['wiki']['enabled'] ) {
 			return null;
 		}
-		return new TwigPageLoader( $provider );
+		return new TwigPageLoader( $provider, $this->config['loaders']['wiki']['rawpages'] ?? [] );
```

Comment:
Hm, can we be sure this field always exists? One might overwrite the settings in `config.prod.json` and break this, because "I don't want no raw pages here".